tests/tensor_search/test_search.py

```diff
@@ -1372,3 +1376,74 @@ def run():
             return True
 
         assert run()
+
+    def test_determine_text_query_prefix(self):
+        """
+        Ensures proper priority order is followed when determining the query prefix.
+        search request-level > index override-level > model default level
+        """
+
+        marqo_index_with_model_default = UnstructuredMarqoIndex(
```

Comment:
I don't think we should be creating indexes directly like this, as it makes cleanup unpredictable as well.
1. Instead, use `cls.create_indexes` and create all your indexes in `setUpClass`. 
2. Also, `tests/tensor_search/test_search.py` is skipped, afaik. This should be in `test_search_combined` under integ tests.

Reply:
Apologies, this should have been deleted in the last round. It's updated counterpart is test_determine_text_chunk_prefix in test_prefix.py